src/document/DocumentCommandHandlers.js

```diff
@@ -848,6 +859,19 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+    /**
+     * Checks and handles if auto update is currently in progress
+     */
+    function checkIfAutoUpdateInProgress() {
+        brackets.app.isAutoUpdateInProgress(function (err, isAutoUpdateInProgress) {
+            if (err === 0 && isAutoUpdateInProgress) {
```

Comment:
No need for an explicit check, general rule we follow in `Brackets` is to check for error condition first. So your statements should be

```

if (err) {
 
} else (isAutoUpdateInProgress) {

}

```

Reply:
Addressed this in 537430b